bindings/java/src/lib.rs

```diff
@@ -45,7 +54,29 @@ pub extern "system" fn Java_org_apache_opendal_Operator_getOperator(
 
     let map = convert_map(&mut env, &params);
     if let Ok(operator) = build_operator(scheme, map) {
-        Box::into_raw(Box::new(operator)) as jlong
+        let java_vm = Arc::new(env.get_java_vm().unwrap());
+        let runtime = Builder::new_multi_thread()
```

Comment:
It's not a good idea to me that building a new runtime for every operator. Do we have a better choice?